### DISCUSSION TITLE
Get 'all' collection.

### DISCUSSION THREAD

**Discussion by sjcobb2022** [Shopify/storefront-api-feedback#175] [Help]
Hi there,

I've been developing a wrapper around the storefront API for a project and I was wondering if I could get any assistance in relation to querying the `all` collection. I want my API to reflect Shopify's own collection filtering system, as using a [ProductFilter] is significantly more convenient and more portable that using queries. 

However, my key issue is not being able to query the `all` collection. Whether it be through querying `collections` or finding a way to parse the filters myself, I have been unable to successfully get a `{url}/collections/all` running in any way using the graphQL api.

Would there be any way to query this data, or would it just be easier to get all collections and then remove any repeating products?

Any help would be more progress than I have made!

**Comment by pearlkrishn** [comment]
@sjcobb2022 I wouldn’t recommend creating a collection page with ‘All’ and adding all products into this collection.
